softbinding-algorithm-entry-schema.json

```diff
@@ -1,110 +1,114 @@
 {
     "$schema": "https://json-schema.org/draft/2020-12/schema",
-    "type": "object",
-    "properties": {
-        "identifier": {
-            "type": "integer",
-            "minimum": 0,
-            "maximum": 65535,
-            "description": "This identifier will be assigned when the soft binding algorithm is added to the list."
-        },
-        "deprecated": {
-            "type": "boolean",
-            "default": false,
-            "description": "Indicates whether this soft binding algorithm is deprecated. Deprecated algorithms shall not be used for creating soft bindings.  Deprecated algorithms may be used for resolving soft bindings but this behaviour is discouraged."
-        },
-        "alg": {
-            "type": "string",
-            "pattern": "(c2pa\\.|[A-Za-z0-9\\-\\.]+)",
-            "description": "Entity-specific namespace as specified for C2PA Assertions labels that shall begin with the Internet domain name for the entity similar to how Java packages are defined (e.g., `com.example.algo1`, `net.example.algos.algo2`)"
-        },
-        "type": {
-            "type": "string",
-            "enum": [
-                "watermark",
-                "fingerprint"
-            ],
-            "description": "Type of soft binding implemented by this algorithm."
-        },
-        "decodedMediaTypes": {
-            "type": "array",
-            "minItems": 1,
-            "items": {
+    "$id": "https://c2pa.org/schemas/softbinding-algorithm-list.json",
+    "type": "array",
```

Comment:
actually the schema was meant to be of type object because it is the schema of an *entry*
softbinding-algorithm-entry-schema.json

is this an issue for the action? As we edit a single list file is it not better to check each entry using the action?

Like:

run: |
          set -e
          echo "Validating each list element…"
          npx jsonschema validate --each softbinding-algorithm-entry-schema.json softbinding-algorithm-list.json
          echo "✓ All elements conform to the entry schema"